build/_build/Build.Steps.cs

```diff
@@ -254,6 +255,52 @@ partial class Build
             CopyFileToDirectory(source, dest, FileExistsPolicy.Overwrite);
         });
 
+    Target DownloadLibSqreen => _ => _
+        .Unlisted()
+        .After(CreateRequiredDirectories)
+        .OnlyWhenStatic(() => IsLinux)
+        .Executes(() =>
+        {
+            var wc = new WebClient();
+            var libSqreenUri = new Uri("https://www.nuget.org/api/v2/package/libsqreen/1.1.2.2");
+            var libSqreenZip = RootDirectory / "libsqreen.zip";
```

Comment:
Can we download this to `TempDirectory` instead?
```suggestion
            var libSqreenZip = TempDirectory / "libsqreen.zip";
```